theia-go-docker/Dockerfile

```diff
@@ -24,9 +24,15 @@ WORKDIR /home/theia
 ADD $version.package.json ./package.json
 RUN yarn --cache-folder ./ycache && rm -rf ./ycache
 RUN yarn theia build
+# See : https://github.com/theia-ide/theia-apps/issues/34
+RUN adduser --disabled-password --gecos '' theia
+RUN chmod g+rw /home && \
+    mkdir -p /home/project && \
+    chown -R theia:theia /home/project;
 EXPOSE 3000
 ENV SHELL /bin/bash
 RUN mkdir /root/.go
```

Comment:
We should use the ```theia``` user's home instead

Reply:
I mean the line ```RUN mkdir /root/.go``` ; the theia user will not be able to use a directory under the root user folder.

Reply:
Ah, sorry, I'll fix this.